app/src/main/res/values-night/color_palette.xml

```diff
@@ -90,4 +90,6 @@
   <color name="color_palette_confetti_red_color">@color/color_def_confetti_rosy_pink</color>
   <color name="color_palette_confetti_yellow_color">@color/color_def_confetti_bright_gold</color>
   <color name="color_palette_confetti_blue_color">@color/color_def_confetti_crystal_blue</color>
+  <color name="color_palette_forward_arrow_button_color">@color/color_def_oppia_turquoise</color>
```

Comment:
Please create this color in `values/color_palette.xml` too.

Reply:
> 

Okay sure